chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -553,27 +553,33 @@ In the next part of the experiment, we will use accuracy to evaluate the model
 This is equal to the ratio between the number of correct predictions a nd the total number of predictions.
 -->
 
-*dịch đoạn phía trên*
+Sau khi huấn luyện mô hình hồi quy softmax, với các đặc trưng làm đầu vào bất kì, chúng ta có thể dự đoán xác suất đầu ra ứng với mỗi lớp. 
+Thông thường, chúng ta sử dụng lớp với xác suất dự đoán cao nhất làm lớp đầu ra. 
+Một dự đoán được xem là chính xác nếu nó trùng khớp hay tương thích với lớp thật sự (nhãn).
+Ở phần tiếp theo của thí nghiệm, chúng ta sẽ sử dụng độ chính xác để đánh giá chất lượng của mô hình. 
```

Comment:
Mình có comment ở thread khác là nên cân nhắc cách dịch chữ accuracy để phân biệt với chữ precision.

Reply:
Từ `accuracy` vẫn được dịch là `độ chính xác` trong nhiều tài liệu rồi. Còn từ `precision` trong `precision and recall` thì trong cuốn Machine Learning Yearning bọn mình giữ nguyên.